exercises/beer-song/canonical-data.json

```diff
@@ -10,32 +10,37 @@
           "cases": [
             {
               "description": "first generic verse",
-              "property": "verse",
-              "number": 99,
+              "property": "recite",
+              "start verse": 99,
+              "end verse": 99,
```

Comment:
Consider `startBottles` and `endBottles` instead.  Same for all occurrences later in this files.